src/cogs/fun.py

```diff
@@ -10,50 +10,29 @@ def __init__(self, bot):
     @nextcord.slash_command()
     async def bofh(self, inter: nextcord.Interaction):
         """Convenient solutions to inconvenient tech problems"""
-        # https://stackoverflow.com/questions/14924721/how-to-choose-a-random-line-from-a-text-file#14924739
-        line_num = 0
-        selected = ""
-        with open("data/excuses.txt") as f:
-            while 1:
-                line = f.readline()
-                if not line:
-                    break
-                line_num += 1
-                if random.uniform(0, line_num) < 1:
-                    selected = line
-        await inter.send(selected.strip())
+        # Wow that was really overcomplicated.
+        lines = open('data/excuses.txt').read().splitlines(
```

Comment:
As noted in the [comment](https://stackoverflow.com/questions/14924721/how-to-choose-a-random-line-from-a-text-file#14924739), this was done with readline for memory/performance reasons. I'll do some checks to see if this actually makes any difference.

Reply:
i don't think it does, but ok.

Reply:
Performance checked! Your code shall stay, nice work.